src/Contact/Avatar.php

```diff
@@ -263,7 +263,7 @@ private static function deleteCacheFile(string $avatar)
 	{
 		$localFile = self::getCacheFile($avatar);
 		if (!empty($localFile)) {
-			unlink($localFile);
+			@unlink($localFile);
```

Comment:
I'd catch such exceptions instead of suppressing it

Reply:
This is just to oppress this message that can occur during deletion when the file had been deleted via some other process after their existence had been checked in this process. There is nothing to handle nor anything to log here. So we could blow it up in some `try` ... `except` ... thingy without anything in the `except` part - or simply that `@`.